controllers/storagecluster/cephfilesystem.go

```diff
@@ -121,11 +122,33 @@ func (obj *ocsCephFilesystems) ensureCreated(r *StorageClusterReconciler, instan
 				return reconcile.Result{}, err
 			}
 		}
+		// create default csi subvolumegroup for the filesystem
+		err = r.createDefaultCsiSubvolumeGroup(cephFilesystem.Name, cephFilesystem.Namespace)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
 	}
 
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) createDefaultCsiSubvolumeGroup(filesystemName, filesystemNamespace string) error {
+	objectMeta := metav1.ObjectMeta{Name: "csi", Namespace: filesystemNamespace}
```

Comment:
Does the name need to be fixed as "csi"? If not we should be generated the name from the filesystem name by appending, subvolumegroup at the end.

Reply:
It would be fixed at is default

Reply:
we dont have any problem this is today as we have single filesystem and we need the SVG name with `csi`